cobrand_hackney/static/app.scss

```diff
@@ -98,6 +98,49 @@ dl.inline {
     list-style-type: none;
 }
 
+.case-detail__map {
+    display: none;
+    height: 400px;
+    @include govuk-media-query($from: tablet) {
+        display: block;
+    }
+    #leaflet {
+        height: 100%;
+    }
+    * + * { margin-top: 0; } // XXX
+}
+.leaflet-marker-icon {
+    cursor: grab;
+}
+
+.case-detail__map__button {
+    @include govuk-media-query($from: tablet) {
+        display: none;
+    }
+}
+
+.case-detail__map__overlay {
+  touch-action: none;
+  display: none;
+  background: hsla(0, 0%, 0%, 0.33);
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  overflow: auto;
+  z-index: 1;
+  margin-top: 0;
+  * + * { margin-top: 0; } // XXX
+}
+.lbh-dialog {
+    box-sizing: border-box;
+  margin: 1em auto;
+  background: white;
+  padding: 2rem;
```

Comment:
Reducing this to `1rem`, and adding `.case-detail__map__overlay .case-detail__map { margin: 1rem 0; }` does a world of wonders here, reducing the whitespace on narrow screens.

Need to decide whether the benefit of consistency from using the LBH dialog element here, overrides the benefit of having a larger, edge-to-edge map. Something to think about later. For now, slightly smaller padding is a step in the right direction.